lib/components/navbar-brand.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+     <b-link v-if="isLink"
+             class="navbar-brand"
+             v-bind="linkProps"
+             @click="$emit('click', $event)">
```

Comment:
Once we hit v2.4 as a peer dependency with Vue, we can start using `v-bind="$attrs" v-on="$listeners"` to proxy everything off to link or whatever. And that is glorious!